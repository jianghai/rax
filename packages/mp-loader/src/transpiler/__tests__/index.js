const transpiler = require('..');

describe('transpiler', () => {
  it('transform', () => {
    const content = `
      <view></view>
    `;
    const result = transpiler(content, {
      templatePath: __filename,
    });

    expect(result).toMatchSnapshot();
  });
});
