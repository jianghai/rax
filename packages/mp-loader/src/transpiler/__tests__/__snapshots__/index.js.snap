// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`transpiler transform 1`] = `
Object {
  "ast": Object {
    "attrsList": Array [],
    "attrsMap": Object {},
    "children": Array [],
    "parent": undefined,
    "plain": true,
    "tag": "view",
    "tagHelperMap": Object {
      "view": "view",
    },
    "type": 1,
  },
  "dependencies": Array [],
  "renderFn": "_c(__components_ref__['view']||'view')",
  "tplASTs": null,
  "tplAlias": null,
}
`;
